js/ui/drop_down_box.d.ts

```diff
@@ -79,10 +79,10 @@ export type OpenedEvent = EventInfo<dxDropDownBox>;
 export type OptionChangedEvent = EventInfo<dxDropDownBox> & ChangedOptionInfo;
 
 /** @public */
-export type PasteEvent = NativeEventInfo<dxDropDownBox>;
+export type PasteEvent = NativeEventInfo<dxDropDownBox, ClipboardEvent>;
 
 /** @public */
-export type ValueChangedEvent = NativeEventInfo<dxDropDownBox> & ValueChangedInfo;
+export type ValueChangedEvent = NativeEventInfo<dxDropDownBox, KeyboardEvent | MouseEvent | PointerEvent | UIEvent> & ValueChangedInfo;
```

Comment:
```suggestion
export type ValueChangedEvent = NativeEventInfo<dxDropDownBox, KeyboardEvent | MouseEvent | PointerEvent | Event> & ValueChangedInfo;
```
https://github.com/DevExpress/DevExtreme/pull/19950/files#r741000230